portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+public class Comment{
+    private final long id;
+    private final String text;
+    private final long timestamp;
+
+    public Comment(long id, String text, long timestamp){
+        this.id=id;
+        this.text=text;
+        this.timestamp=timestamp;
+
```

Comment:
remove newline.

Reply:
I see that should all be fixed now in the new commit